.eslintrc.js

```diff
@@ -5,13 +5,12 @@ module.exports = {
   parser: 'babel-eslint',
   extends: [
     'eslint:recommended',
-    'prettier',
     'plugin:import/errors',
     'plugin:qunit/recommended',
+    'plugin:prettier/recommended',
```

Comment:
This is the recommended configuration https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

It does three things..

- Enables `eslint-plugin-prettier`.
- Sets the `prettier/prettier` rule to `"error"`.
- Extends the `eslint-config-prettier` configuration.